pyaedt/modeler/Primitives3DLayout.py

```diff
@@ -232,6 +236,156 @@ def objects_by_net(self, net_name, object_filter=None, include_voids=False):
             objs = self.modeler.oeditor.FindObjects("Net", net_name)
         return objs
 
+    @pyaedt_function_handler()
+    def _get_names(self, categories):
+        names = []
+        for object in categories:
+            names.extend(self.modeler.oeditor.FindObjects("Type", object))
```

Comment:
This should fix the codacy issue.
```suggestion
        for category in categories:
            names.extend(self.modeler.oeditor.FindObjects("Type", category))
```